SwappinessCheck.pm

```diff
@@ -35,33 +35,41 @@ sub execute
 	$self->{RESULT} = "";
 	$self->{COMMENT} = undef;
 
-	my $sysctlcmd;
-	if (-e "/sbin/sysctl") {
-		$sysctlcmd="/sbin/sysctl"
-	} else {
-		$sysctlcmd="sysctl"
+	my $swapon=`swapon -s`;
+	if ($swapon eq "")
+	{
+		$self->{RESULTKIND} = "good";
```

Comment:
Maybe add a `$self->{COMMENT} = "swap is not enabled"`?

Reply:
Thank you @raboof .
I have rather the idea to let the user the choice not to use the swap for his system. Let me explain:
- it being established that the absence of swap is not detrimental to real-time audio as long as the system is not running out of RAM,
- being established that, if the system user or his administrator decides to use the swap, it is absolutely necessary to set the triggering of its use when it is 90% full to avoid jitter,
then the test should be run in this sense and return the "good" result by giving an indication that would rather encourage the removal of swap on an audio system instead of leaving the user in doubt as to whether he should put one in place.
That said, I will open a new pull request for that.

[edit] Oups, rereading your comment again, I realize that you think exactly the same as me or at least that your addition would be completely in line with what I just wrote! :-) will do soon.